src/test/java/com/easypost/BatchTest.java

```diff
@@ -17,22 +17,24 @@
 import static org.junit.jupiter.api.Assertions.assertNotNull;
 import static org.junit.jupiter.api.Assertions.assertTrue;
 
-public class BatchTest {
+public final class BatchTest {
+
```

Comment:
Is Checkstyle enforcing this line break here? Not a huge fan.

Reply:
No, CheckStyle doesn't rule one way or the other on that, just a stray line break